File: dataset/SemanticCloneBench/python/Clone96.py
def SumOdds(x, y) :
	count = 0
	for i in range(x, y + 1) :
		if i % 2 == 1 :
			count = count + i
	print (count)


def SumOdds(x, y) :
	count = 0
	for i in range(x, y) :
		if (int(i % 2 == 1)) :
			count = count + i
	if (x % 2 == 0) :
		count = count + x
	if (y % 2 == 0) :
		count = count + 7
	print (count)

File: dataset/SemanticCloneBench/cs/Clone96.cs
/*
* Semantic clone benchmark
*  Source code are extracted from Stack Overflow
*  Stack overflow Question #:30633258
*  Stack Overflow answer #:30633371
*  And Stack Overflow answer#:30633565
*/
private static void Main (string [] args) {
    const string input = @"[Testing.User]|Info:([Testing.Info]|Name:([System.String]|Matt)|Age:([System.Int32]|21))|Description:([System.String]|This is some description)";
    const string pattern = @"(\[Testing\.User\])\|(Info:.*)\|(Description:.*)";
    var match = Regex.Match (input, pattern);
    if (match.Success) {
        for (int i = 1; i < match.Groups.Count; i ++) {
            Console.WriteLine ("[" + i + "] = " + match.Groups [i]);
        }
    }
    Console.ReadLine ();
}

public static void Main () {
    String text = "[Testing.User]|Info:([Testing.Info]|Name:([System.String]|Matt)|Age:([System.Int32]|21))|Description:([System.String]|This is some description)";
    String pattern = "(\\[\\w+\\.\\w+\\])\\|(\\w+:.+)\\|(\\w+:.+)";
    Match match = Regex.Match (text, pattern);
    if (match.Success) {
        Console.WriteLine (match.Groups [1]);
        Console.WriteLine (match.Groups [2]);
        Console.WriteLine (match.Groups [3]);
    }
}

